tasks/cleanup.yml

```diff
@@ -1,27 +1,34 @@
 ---
 # tasks file for cleanup
-
 - name: combine existing ownerRefences with new ownerReferences
   set_fact:
-    _owner_references: "{{ item.metadata.ownerReferences | default([]) + owner_references | unique | list }}"
+    _owner_references: "{{ item.metadata.ownerReferences | default([]) + (cluster_resources.resources[0].metadata.ownerReferences | default([]) ) + owner_references | unique | list }}"
```

Comment:
moet dit niet hetzelfde blijven?
cluster_resources wordt hieronder pas aangemaakt

Reply:
@Shalucik klopt! ik heb dhet combinen later opgesplitst in meerdere taken voor betere leesbaarheid. Ik heb het aangepast.